index.ios.js

```diff
@@ -1,77 +0,0 @@
-/**
- * @providesModule MusicControl
- * @flow
- */
-'use strict';
-
-import { NativeModules, NativeEventEmitter } from 'react-native';
-const NativeMusicControl = NativeModules.MusicControlManager;
-
-/**
- * High-level docs for the MusicControl iOS API can be written here.
- */
-var handlers = { };
-var subscription = null;
-
-var MusicControl = {
-
-  STATE_PLAYING: NativeMusicControl.STATE_PLAYING,
-  STATE_PAUSED: NativeMusicControl.STATE_PAUSED,
-  STATE_ERROR: NativeMusicControl.STATE_PAUSED,
-  STATE_STOPPED: NativeMusicControl.STATE_PAUSED,
-  STATE_BUFFERING: NativeMusicControl.STATE_PAUSED,
```

Comment:
I don't think merging index.ios.js and index.android.js is a good idea. Have you tested everything to make sure it works?
For instance, the constants in iOS and Android are defined in a different way, as you can see here.